packages/ast/template-tag/tests/helpers/test-on-posix.ts

```diff
@@ -0,0 +1,11 @@
+import { EOL } from 'node:os';
+
+import { test } from '@codemod-utils/tests';
+
+const onPosix = EOL === '\n';
+
+export function testOnPosix(...parameters: Parameters<typeof test>): void {
```

Comment:
This test helper helps us skip tests that need a complex conditional branch to pass on Windows.